### ISSUE TITLE
Routed parent/child component communication not working

### ISSUE THREAD

**Issue by aestheticsdata** [angular-ui/ui-router#3277]
Using ui-router RC1,

does the child need to have a child state of the parent in order to get the foo-binding on the <div ui-view foo-binding="$ctrl.bar"> ?

Is it possible with:
```
parent : .state('hello, ...
child : .state('world', ...
```

or do we need to have:
```
parent: .state('hello', ...
child: .state('hello.world', ...
```
I have `<div ui-view foo-binding="$ctrl.bar">` on my root component template, but the routed child is not receiving the fooBinding.

**Comment by christopherthielen** [comment]
It should send the binding down through the `ui-view` as long as it's routing to a component inside the `ui-view`. 

Can you show me a simple plunker that demonstrates the issue?

http://plnkr.co/edit/xJD0qgct6u8PaIjQ1LWc?p=info

**Comment by aestheticsdata** [comment]
Here is the plunkrs : 

in this case it works : http://plnkr.co/edit/JI7BtA2uhW1t1txFanYp
the bindings (fooBar and myCoolFunction) from the div of the rootobj are sent to the login component.

but if there is an intermediate state with a redirect it's not working anymore : http://plnkr.co/edit/CLpaq25Fe1ZagNbgvKBa
in this case the bindings are not available in the login component

**Comment by kalbert312** [comment]
I just ran into this exact issue. 
I have a non-abstract intermediate state that is just `<ui-view></ui-view>` so I can have a UI tab in my application that 

1. Points to an intermediate state, that way I can utilize `ui-sref-active` for the tab UI so that any child state marks that tab as active.
2. Redirects to a landing state for that tab.

I tried making the intermediate state abstract with a redirect but this does not seem to work. When I click a link to this abstract state, it does not redirect, it just throws an error that I cannot navigate to an abstract state.

If I remove the intermediate state, which is really only there for the purposes of using `ui-sref-active` on my tab, then it works.
